src/core/atomicdex/services/mm2/mm2.service.cpp

```diff
@@ -641,8 +641,17 @@ namespace atomic_dex
                     request.address_format                   = nlohmann::json::object();
                     request.address_format.value()["format"] = "segwit";
                 }
+
+                if (coin_info.utxo_merge.value_or(false))
+                {
+                    t_utxo_merge_params params{.merge_at = 300, .check_every = 60, .max_merge_at_once = 200};
```

Comment:
we would have to test if it works with max_merge_at_once=200, i can imagine it would cause issues with some electrums, so maybe it's better to set max_merge_at_once to 100
check_every = 60 means it checks every minute... do we need to do this every minute? maybe better to do it every 10 minutes or so
any maybe also better to start earlier, not wait for 300 utxos
what do you guys think?

Reply:
I"m happy to tweak the default params - just used those to drop around 3500 RICK utxos into 130 or so relatively quickly.
There is a user with 23000 utxos from mining tokel, which will take a while to consolidate either way.

From memory, up to 800 or so utxos can be merged in a tx.
I"m not sure at what number utxos electrums begin to complain about "payload too large"
I think every 5 min is a balance between not too obtrusive, and not too slow.

Reply:
OK, sounds like it will work... there is a difference though, RICK (and all other coins run by me) electrums have `MAX_SEND = 2000000` set, the default is 1M

Reply:
updated to use these values:
merge_at = 250
check_every = 300
max_merge_at_once = 125